### DISCUSSION TITLE
seems there is security issue for docker image or project!!

### DISCUSSION THREAD

**Discussion by weiminw** [open-webui/open-webui#6937] [General]
When I start the project in docker on internet. I found this user always show up  and attempt to register many many time whenever and where ever I start up .

<img width="1131" alt="截屏2024-11-14 上午11 31 13" src="https://github.com/user-attachments/assets/9365f16a-7cac-4be3-b1ff-8f606ea9cdbd">

**Comment by tjbck** [comment]
New signups will be disabled by default from subsequent releases.
